vaadin-list-mixin.html

```diff
@@ -169,16 +169,23 @@
       const condition = item => !(item.disabled || this._isItemHidden(item));
       let idx, increment;
 
+      const hostIsRTL = this.getAttribute('dir') === 'rtl';
```

Comment:
Consider using the same code as in vaadin/component-mixins#33

```js
const isRTL = !this._vertical && this.getAttribute('dir') === 'rtl';
const dirIncrement = isRTL ? -1 : 1;
```

Reply:
Done.